# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1338906277 -7200
#      Tue Jun 05 16:24:37 2012 +0200
# Branch gui
# Node ID ddd9c2a91de575d9ed0d31db444cd1c8ed15ed90
# Parent  f349846566588fe0be9d954c69632ac1a8d20a49
Adjusted file browser up directory icon.

* files-dockwidget.cc: Adjusted icon.

diff --git a/gui/src/files-dockwidget.cc b/gui/src/files-dockwidget.cc
--- a/gui/src/files-dockwidget.cc
+++ b/gui/src/files-dockwidget.cc
@@ -34,19 +34,17 @@ files_dock_widget::files_dock_widget (QW
 
   // Create a toolbar
   _navigation_tool_bar = new QToolBar ("", widget ());
   _navigation_tool_bar->setAllowedAreas (Qt::TopToolBarArea);
   _navigation_tool_bar->setMovable (false);
   _navigation_tool_bar->setIconSize (QSize (20, 20));
 
   // Add a button to the toolbar with the QT standard icon for up-directory
-  // TODO: Maybe change this to be an up-directory icon that is OS specific???
-  QStyle *style = QApplication::style ();
-  _directory_icon = style->standardIcon (QStyle::SP_FileDialogToParent);
+  _directory_icon = QIcon(":/actions/icons/up.png");
   _directory_up_action = new QAction (_directory_icon, "", _navigation_tool_bar);
   _directory_up_action->setStatusTip (tr ("Move up one directory."));
 
   _current_directory = new QLineEdit (_navigation_tool_bar);
   _current_directory->setStatusTip (tr ("Enter the path or filename."));
 
   _navigation_tool_bar->addAction (_directory_up_action);
   _navigation_tool_bar->addWidget (_current_directory);
